core/src/test/java/io/questdb/cutlass/pgwire/PGJobContextTest.java

```diff
@@ -6253,6 +6274,143 @@ public int read() {
         });
     }
 
+    private void testBindVariableIsNull(boolean binary) throws Exception {
+        assertMemoryLeak(() -> {
+            try (
+                    final PGWireServer ignored = createPGServer(1);
+                    final Connection connection = getConnection(false, binary)
+            ) {
+                connection.setAutoCommit(false);
+                connection.prepareStatement("create table tab1 (value int, ts timestamp) timestamp(ts)").execute();
+                connection.prepareStatement("insert into tab1 (value, ts) values (100, 0)").execute();
+                connection.prepareStatement("insert into tab1 (value, ts) values (null, 1)").execute();
+                connection.commit();
+
+                sink.clear();
+                try (PreparedStatement ps = connection.prepareStatement("tab1 where null is ?")) {
```

Comment:
My suggestion was for `? is [not] null` (so, left hand of the operator), not `42 is ?` / `col is ?`. Are there any SQL database that support syntax from this test? My impression is that IS NULL always assumes `expression IS NULL` syntax and we should implement it in this way. Expression here may be a constant, a column, a bind variable, or an expression (e.g. `col + 42`).

Reply:
oh!! I misunderstood, indeed null not as a wild, will fix, thank you!